pkg/reading/status.go

```diff
@@ -0,0 +1,10 @@
+package reading
+
+type Status struct {
+	Id          string `json:"id"`
+	Name        string `json:"name"`
+	Description string `json:"description"`
+	IconUrl     string `json:"iconUrl"`
+}
+
+type Statuses []Status
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type Statuses should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4860757)